CategoryContentLegacyEvent.php

```diff
@@ -196,7 +194,7 @@ public function onAdminProductCategoryEditAfter()
             ->find($id);
 
         if (is_null($CategoryContent)) {
-            $CategoryContent = new \Plugin\CategoryContent\Entity\CategoryContent();
+            $CategoryContent = new CategoryContent();
         }
 
         if ('POST' === $app['request']->getMethod()) {
```

Comment:
Duplicate check in 
https://github.com/eccubevn/category-content-plugin/pull/1/commits/93960cabbb932ce46df62b56e4bf2692d4299640#diff-aa4110264c5d6e1e4404f0d9fe732527R183

Change $app['orm.em']->flush(); to $app['orm.em']->flush($CategoryContent);